lib/std/build.zig

```diff
@@ -1395,6 +1397,8 @@ pub const LibExeObjStep = struct {
         });
     }
 
+    /// Override the build Target.
+    /// The default target is the host system.
```

Comment:
This is incorrect when using `standardTargetOptions`, which should be noted here